### ISSUE TITLE
Can't Set Ref on Props in React v15

### ISSUE THREAD

**Issue by hew** [gilbox/react-track#14]
Trying to run this component with React v15 results in this error.

```
Warning: Element: `ref` is not a prop. Trying to access it will result in `undefined` being returned. If you need to access the same value within the child component, you should pass it as a different prop. (https://fb.me/react-special-props)
```

See [this area](https://github.com/gilbox/react-track/blob/master/src/index.js#L22) of the code.

**Comment by gilbox** [comment]
Thanks for reporting! The `ref` prop will need to be renamed. Maybe something like `trackedRef`

**Comment by hew** [comment]
No prob. I tested your suggestion: simply subbing 'trackedRef' for 'ref. Seems to work just fine. 

[Pull Request —>](https://github.com/gilbox/react-track/pull/18)

**Comment by gilbox** [comment]
Thanks for the PR! Released on NPM `react-track@0.4.0`.
